tests/check/check_signed_video_auth.c

```diff
@@ -1431,6 +1431,40 @@ START_TEST(file_export_with_two_useless_seis)
 }
 END_TEST
 
+/* Test description
+ * Verify that we do not get any authentication if the stream has onvif SEIs.
+ */
+START_TEST(onvif_seis)
+{
+  // ONVIF SEI is not supported for AV1, so skip the test in that case
```

Comment:
I think you should use the real name "ONVIF Media Signing is not supported for AV1"